src/ceph-disk/ceph_disk/main.py

```diff
@@ -3243,9 +3243,10 @@ def activate(
             else:
                 init = init_get()
 
-        LOG.debug('Marking with init system %s', init)
-        with file(os.path.join(path, init), 'w'):
-            pass
+    LOG.debug('Marking with init system %s', init)
+    init_path = os.path.join(path, init)
+    with file(init_path, 'w'):
+        path_set_context(init_path)
```

Comment:
it should be indented to the right so that it is included in the if instead of being outside. Otherwise looks good and ready for tests ;-)